contracts/sg-ics721/src/execute.rs

```diff
@@ -48,7 +48,9 @@ impl Ics721Execute for SgIcs721Contract {
                 // therefore, we use ics721 creator as owner
                 creator: ics721_contract_info.creator,
                 description: "".to_string(),
-                image: "https://arkprotocol.io".to_string(),
+                // use Stargaze icon as placeholder
+                image: "ipfs://bafkreie5vwrm5zts4wiq6ebtopmztgl5qzyl4uszyllgwpaizyc5w2uycm"
```

Comment:
Maybe this constant should be in a variable somewhere or in an external file?

Reply:
moved to `STARGAZE_ICON_PLACEHOLDER`